www/src/data/sidebars/doc-links.yaml

```diff
@@ -83,19 +83,30 @@
           link: /docs/programmatically-create-pages-from-data/
         - title: Querying data with StaticQuery
           link: /docs/static-query/
-    - title: Headless CMS*
-      link: /docs/headless-cms/
+    - title: Sourcing content and data*
```

Comment:
This whole section should be moved before the querying. We should be putting these sections in the order the user would be doing them, similar to the order in https://www.gatsbyjs.com/integrations/ or https://www.gatsbyjs.com/how-it-works/

Reply:
(this could be done in a separate PR as well, obviously there's more re-ordering)

Reply:
Ok! @KyleAMathews initially helped put these in order, so we'll need to sync up about this!

Reply:
oh wait, I was looking at the wrong section header for this comment. So, because Headless CMSs is within this category, wouldn't someone need to know how to query with GraphQL first before reading about Headless CMSs?

Reply:
It's not a big deal one way or the other bug I agree with Sam here that someone new to Gatsby would first want to read a bit about how you can source data from anywhere before they move onto using graphql to pull that data into their components.